lisa/sut_orchestrator/azure/features.py

```diff
@@ -1749,40 +1794,69 @@ def get_luns(self) -> Dict[str, int]:
                 device_luns.update({cmd_result.stdout: device_lun})
         return device_luns
 
-    def get_raw_data_disks(self) -> List[str]:
-        # Handle BSD case
-        if isinstance(self._node.os, BSD):
-            return self._get_raw_data_disks_bsd()
-
-        # disk_controller_type == NVME
+    def get_nvme_os_disk_controller(self) -> str:
```

Comment:
Reused the Nvme feature, not to copy code.

Reply:
Fixed